src/context.rs

```diff
@@ -0,0 +1,107 @@
+use crate::{InternalError, SignatureError};
+
+/// Ed25519 contexts as used by Ed25519ctx and Ed25519ph.
```

Comment:
I wasn't sure if it makes sense to mention Ed25519ctx here as it's a building block and not directly implemented by this crate.